contracts/treasury/GovernorCountingOverridable.sol

```diff
@@ -181,13 +181,20 @@ abstract contract GovernorCountingOverridable is Initializable, GovernorUpgradea
         uint256 timepoint = proposalSnapshot(_proposalId);
         address delegate = votes().delegatedAt(_account, timepoint);
 
-        bool isTranscoder = _account == delegate;
-        if (isTranscoder) {
-            // deduce weight from any previous delegators for this transcoder to
-            // make a vote
+        bool selfDelegates = _account == delegate;
+        if (selfDelegates) {
+            // deduce weight from any previous delegators for this self-delegating account to cast a vote
             return _weight - _voter.deductions;
         }
 
+        address delegatesDelegate = votes().delegatedAt(delegate, timepoint);
```

Comment:
If this var is not used outside of the next line, moving the call into the next line might help with readability since there isn't a need to come up with a good name for this var:

```
bool isDelegateSelfDelegated = delegate == votes().delegatedAt(delegate, timepoint)
```